docs/rules/space-before-keywords.md

```diff
@@ -30,75 +30,75 @@ the keywords `else`, `while` (do...while), `finally` and `catch`. The default va
 This rule will allow keywords to be preceded by an opening curly brace (`{`). If you wish to alter
 this behaviour, consider using the [block-spacing](block-spacing.md) rule.
 
-The following patterns are considered errors when configured `"never"`:
+Examples of **incorrect** code for this rule with the default `"always"` option:
 
 ```js
-/*eslint space-before-keywords: ["error", "never"]*/
+/*eslint space-before-keywords: ["error", "always"]*/
+/*eslint-env es6*/
 
 if (foo) {
     // ...
-} else {}
-
-do {
+}else {}
 
-}
-while (foo)
+const foo = 'bar';let baz = 'qux';
 
-try {} finally {}
+var foo =function bar () {}
 
-try {} catch(e) {}
+function bar() {
+    if (foo) {return; }
+}
 ```
 
-The following patterns are not considered errors when configured `"never"`:
+Examples of **correct** code for this rule with the default `"always"` option:
 
 ```js
-/*eslint space-before-keywords: ["error", "never"]*/
+/*eslint space-before-keywords: ["error", "always"]*/
+/*eslint-env es6*/
 
 if (foo) {
     // ...
-}else {}
+} else {}
 
-do {}while (foo)
+(function() {})()
```

Comment:
Yeah, sorry it seems a weird change but the reason is that I've moved the structure of the examples to show the default option (`"always"`) first because I've seen that other rules follow this order and I think it's better. I've tried to explain it at the PR description but maybe the explanation was not the best, sorry. I've created a mix of screenshots to show the changes made here, but the idea is to change the position of two example blocks putting first the default option and then the other one.

Thanks for the support

![commit_explanation](https://cloud.githubusercontent.com/assets/3751801/20597477/33673fc4-b244-11e6-8f42-00c48ca21212.png)